docs/source/getting_started/content/con_understanding-codeready-containers-configuration.adoc

```diff
@@ -0,0 +1,10 @@
+[id="understanding-codeready-containers-configuration_{context}"]
+= Understanding {prod} configuration
```

Comment:
As a general rule, I stay away from "Understanding" as a title. There was a discussion in the Style council about the best way to title topics like this, but you can never guarantee understanding. "About {prod} configuration" is more accurate, or "{prod} configuration explained" (Which I don't like the sound of, but is more accurate than 'understanding')

Reply:
Fair enough. In this case, I think "About {prod} configuration" is a suitable title.